docarray/array/mixins/io/binary.py

```diff
@@ -49,6 +49,8 @@ def load_binary(
                 protocol=protocol,
                 compress=compress,
                 _show_progress=_show_progress,
+                *args,
+                **kwargs,
```

Comment:
I don't agree on this
Actually we can't really stream a storage document array right ?